src/features/pastelDB/pastelDBLib.ts

```diff
@@ -109,3 +125,408 @@ export const createTables = async (db: Database): Promise<void> => {
   db.exec(create_listaddresses)
   return
 }
+
```

Comment:
Could you try to add tests to this new functions, please? They seems to be a perfect candidate. Here is an example: https://github.com/pastelnetwork/pastel-electron-wallet/blob/master/src/features/loading/__tests__/utils.test.ts

Reply:
- Added tests